libraries/Microsoft.Bot.Builder/Teams/TeamsActivityExtensions.cs

```diff
@@ -8,6 +8,12 @@ namespace Microsoft.Bot.Builder.Teams
 {
     public static class TeamsActivityExtensions
     {
+        public static string TeamsGetChannelId(this IActivity activity)
+        {
+            var channelData = activity.GetChannelData<TeamsChannelData>();
+            return channelData.Channel.Id;
```

Comment:
We need to do a null check here. The TeamsChannelData object won't exist in group chat or 1:1 chats